app/src/main/res/values/strings.xml

```diff
@@ -151,6 +151,55 @@
     <!-- Item in Preferences that shows the about page. Parameter 1 is the app name, i.e. Firefox Focus/Firefox Klar. -->
     <string name="preference_about">About %1$s</string>
 
+    <!-- Header for the list of installed search engines -->
+    <string name="preference_search_installed_search_engines">Installed search engines</string>
+
+    <!-- Action for restoring the default list of search engines -->
+    <string name="preference_search_restore">Restore default search engines</string>
+
+    <!-- Action for adding an additional search engine to the list -->
+    <string name="preference_search_add">Add another search engine</string>
+    <string name="preference_search_remove">Remove</string>
+
+    <!-- Clickable text in tutorial to open a dialog to manually add a search engine -->
+    <string name="tutorial_search_manually_add">MANUALLY ADD</string>
+    <string name="tutorial_search_cancel">CANCEL</string>
```

Comment:
I wouldn't capitalize the strings here (and further down). For buttons that's done by Android automatically (in languages where it makes sense!) and in other locations we can use the android:capitalize attribute to control it ourselves.

Reply:
Ah, just saw that we also use this string in placeholders. In this case it might be a bit more complicated than I thought - not sure if we want to call uppercase() on the string before replacing the placeholder?

Reply:
Sounds good, I'll capitalize these normally. I also removed Cancel because we already have that as a general string.